### PR TITLE
Improve consistency with @csstools/normalize.css

### PR DESCRIPTION
**Pull request by kripod**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Please fill out the description field when you do pull requests. It's not clear why we should be consistent with a random fork of  https://github.com/necolas/normalize.css/

**Comment by kripod** [comment]
Thanks for the fast feedback and sorry for the lack of description. The reason behind forking the original project is explained [in the changelog](https://github.com/csstools/normalize.css/blob/83a891ae47d6c2ddbb7533238c87646e2783572e/CHANGELOG.md#1000-may-16-2019) of csstools/normalize.css. It isn't just a random fork, as Jonathan Neal, the original author of normalize.css besides Nicolas Gallagher is behind it:

> There are now 2 versions of normalize.css. Nicolas Gallagher and I started
> writing normalize.css together. I named and created the normalize.css
> repository with the help of Paul Irish and Ben Alman. I transferred the
> repository to Necolas, who made it into a “household” CSS library. Much
> later I resumed management of normalize.css with Luciano Battagliero. We
> tagged, deprecated, and removed “opinionated” styles — styles developers
> often prefer but which do not fix bugs or “normalize” browser differences.
> Necolas disagreed with this change, and resolved the matter AFAIK by removing
> all of the other contributors, locking discussion threads, wiping my name
> (and his) from all files, and blocking me from being able to follow the
> project.
>
> I may later create a new project with a new name, but for now I intend to
> continue working on the normalize.css project, sometimes under the
> “csstools” tag. I hope one day our differences are resolved and the projects
> will be one again.
>
> For reference within this project:
> normalize.css resolves bugs and common browser inconsistencies.
> opinionated.css does the same while preserving the classic opinionated styles.

File: modern-normalize.css [hunk 1]
```diff
@@ -204,13 +201,18 @@ button::-moz-focus-inner,
...
-button:-moz-focusring,
-[type='button']:-moz-focusring,
-[type='reset']:-moz-focusring,
-[type='submit']:-moz-focusring {
+:-moz-focusring {
 	outline: 1px dotted ButtonText;
 }
 
+/**
+Remove the additional ':invalid' styles in Firefox.
```
Comment (sindresorhus) [inline modern-normalize.css:209]: Are you sure this is still valid? Can you try to find a Firefox bug for this a link to it in the comment?

**Comment by sindresorhus** [comment]
> The reason behind forking the original project is explained in the changelog of csstools/normalize.css. It isn't just a random fork, as Jonathan Neal, the original author of normalize.css besides Nicolas Gallagher is behind it:

That should really have been in the readme of that repo.

File: modern-normalize.css (hunk 1 above)
Comment (kripod) [reply modern-normalize.css:209]: I've checked the source code of Firefox and can confirm that the `box-shadow` property is [overridden through UA sheets](https://dxr.mozilla.org/mozilla-central/rev/c68fe15a81fc2dc9fc5765f3be2573519c09b6c1/layout/style/res/forms.css#745-754).

More code search results can be found [here](https://dxr.mozilla.org/mozilla-central/search?q=%22%3A-moz-ui-invalid%22&redirect=false).

Comment (sindresorhus) [reply modern-normalize.css:209]: Can you add a link to it in the comment? Using this mirror https://github.com/mozilla/gecko-dev

Comment (kripod) [reply modern-normalize.css:209]: Sure, I've just added it.

**Comment by kripod** [comment]
I totally agree, thanks for the suggestion! I've opened an issue about it in that repo.

**Pull request merged**